kivydesigner/uix/modalmsg.py

```diff
@@ -0,0 +1,36 @@
+from pathlib import Path
+from kivy.uix.modalview import ModalView
+from kivy.lang import Builder
+from kivy.properties import StringProperty, NumericProperty
+
+kv_filepath = Path(__file__).with_suffix('.kv')
+Builder.load_file(str(kv_filepath), rulesonly=True)
+
+class ModalMsg(ModalView):
+    title = StringProperty('Kivy Designer')
+    message = StringProperty('')
+    icon_name = StringProperty('info')
```

Comment:
Add a comment explaining why this works. The icon name has to be registered in `resources.py`

Reply:
Fixed in [803bdac](https://github.com/Joshua-Douglas/kivydesigner/pull/17/commits/803bdac05c273f7db0d0e9b4a6e426af4bf42305).